### PR TITLE
Add boolean for company certification communication

### PR DESCRIPTION
**Pull request by rtaieb**
https://trello.com/c/AiY65dej/1060

### CODE REVIEW

File: app/models/company_certification.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class CompanyCertification(BaseModel):
     company = db.relationship("Company", backref=backref("certifications"))
 
     attribution_date = db.Column(db.Date, nullable=False)
-    expiration_date = db.Column(db.Date, nullable=True)
+    expiration_date = db.Column(db.Date, nullable=False)
```
Comment (rtaieb) [inline app/models/company_certification.py:14]: :orange_circle: A discuter pour voir si c'était à Nullable pour une raison métier

File: app/models/company.py [hunk 2]
```diff
@@ -95,3 +97,16 @@ def retrieve_authorized_clients(self):
...
             )
         )
+
+    @cached_property
+    def is_certified(self):
+        today = date.today()
+        for company_certification in self.certifications:
+            if (
+                company_certification.attribution_date
+                <= today
```
Comment (sandrica89) [inline app/models/company.py:107]: :orange_circle: `company_certification.attribution_date <= today` is still true, isn't it?

File: app/models/company_certification.py (hunk 1 above)
Comment (sandrica89) [reply app/models/company_certification.py:14]: `expiration_date` is not null only if the company is certified

File: app/tests/certificate/test_certificate_company_api.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
+from datetime import timedelta, date, datetime
+
+from app.seed import CompanyFactory
+from app.seed.factories import CompanyCertificationFactory, UserFactory
+from app.tests import BaseTest
+from app.tests.helpers import make_authenticated_request, ApiRequests
+
+EXPECTED_CERTIFICATION_DATE_FORMAT = "%Y/%m/%d"
```
Comment (sandrica89) [inline app/tests/certificate/test_certificate_company_api.py:8]: :green_circle: Not used

**Review by sandrica89 — CHANGES_REQUESTED** [review]

File: migrations/versions/90acbc440024_add_certification_communication_boolean.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        sa.Column(
+            "accept_certification_communication", sa.Boolean(), nullable=True
+        ),
+    )
+    op.alter_column(
+        "company_certification",
+        "expiration_date",
+        existing_type=sa.DATE(),
+        nullable=False,
+    )
```
[the remark this answers is not in the document]
Comment (sandrica89) [reply migrations/versions/90acbc440024_add_certification_communication_boolean.py:31]: Not relevant anymore

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**